servant/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ next
 ----
 
 * Add `CaptureAll` combinator. Captures all of the remaining segments in a URL.
+* replace use of `ToByteString` with `HttpApiData` for `GetHeaders`
```

Comment:
Preface with "BACKWARDS INCOMPATIBLE" (new thing, so isn't in any of the other ones yet.
And now it is "Replace `To/FromByteString` with `To/FromHttpApiData` for `GetHeaders/BuildHeadersTo`"